make.ps1

```diff
@@ -18,17 +18,74 @@ function FindMSBuild
 	return $null
 }
 
+function CopyDependencies
+{
+	cd thirdparty
+	./fetch-thirdparty-deps.ps1
+	cp *.dll ..
+	cp windows/*.dll ..
+	cd ..
+	echo "Dependencies copied."
+
+	return $LASTEXITCODE
+}
+
+function TestModsAndMaps
+{
+	if(Test-Path ./OpenRA.Utility.exe)
+	{
+		echo "Testing mods..."
+		if(Test-Path ./mods/ra/OpenRA.Mods.RA.dll)
+		{
+			echo ""
+			echo "Testing Red Alert mod MiniYAML..."
+			./OpenRA.Utility.exe ra --check-yaml
+	
+			if(Test-Path ./mods/cnc/OpenRA.Mods.Cnc.dll)
+			{
+				echo ""
+				echo "Testing Tiberian Dawn mod MiniYAML..."
+				./OpenRA.Utility.exe cnc --check-yaml
+			}
+
+			if(Test-Path ./mods/d2k/OpenRA.Mods.D2k.dll)
+			{
+				echo ""
+				echo "Testing Dune 2000 mod MiniYAML..."
+				./OpenRA.Utility.exe d2k --check-yaml
+			}
+
+			if(Test-Path ./mods/ts/OpenRA.Mods.TS.dll)
+			{
+				echo ""
+				echo "Testing Tiberian Sun mod MiniYAML..."
+				./OpenRA.Utility.exe ts --check-yaml
+			}
+		}
+		else
+		{
+			echo "Error: Red alert is needed to work properly!"
+			return 1
+		}
+	}
+	else
+	{
+		echo "Error: OpenRA.Utility.exe is missing."
+		return 1
+	}
+}
+
 if ($args.Length -eq 0)
 {
 	echo "Command list:"
 	echo ""
-	echo "  all             Builds the game and its development tools."
+	echo "  all             Builds the game, development tools and copies dependencies."
```

Comment:
Since the dependencies get copied first, you should reorder those to reflect what happens.